src/symbols/Port/Port.tsx

```diff
@@ -145,10 +178,18 @@ export const Port = forwardRef(
           animate={{
             x: newX,
             y: newY,
-            scale: (dragging || active || hovered) && !isDisabled ? 1.5 : 1,
+            scale: (isDragging || active || isHovered) && !isDisabled ? 1.5 : 1,
             opacity: 1
           }}
         />
+
```

Comment:
nit: remove linebreak

Reply:
My IDE is configured to add line breaks at the bottom automatically, I'll try to see if I can disable it for reaflow projet.

Reply:
Ah I misunderstood this, I thought it was an end file blank line.

I find the code more readable when using blank lines between blocks but I'll change it back.